### DISCUSSION TITLE
Add support for custom themes

### DISCUSSION THREAD

**Discussion by hg0428** [thebaselab/codeapp#367] [Ideas]
I am also a code editor designer, except I design online editors. It's not too difficult to add custom theme support, as well as basic settings such as font family, font size, font weight, letter spacing, ligatures, minimap or more.

My suggestion is that you build a settings menu with these options.

**Comment by luni-moon** [comment]
> I am also a code editor designer, except I design online editors. It's not too difficult to add custom theme support, as well as basic settings such as font family, font size, font weight, letter spacing, ligatures, minimap or more.
> 
> 
> 
> My suggestion is that you build a settings menu with these options.

We already have preset themes in a settings menu...

**Reply by hg0428** [reply]
I am asking for custom themes and more settings options

**Comment by bummoblizard** [comment]
That's a great idea. Ideally, it should be compatible with existing vscode themes.

**Reply by hg0428** [reply]
Yes, if the feature was added it would most likely support pre-existing themes because of how the monaco editor is built.
